setup.py

```diff
@@ -114,6 +116,8 @@ def get_requirements():
     if os.name != 'nt':  # no pypi torch for windows
         if os.getenv('PYTORCH_VERSION'):
             requirements.append('torch==%s' % os.getenv('PYTORCH_VERSION'))
+        elif IGN_TORCH_VER:
```

Comment:
Can you also add the same behavior at https://github.com/NVIDIAGameWorks/kaolin/blob/master/setup.py#L25 ? (with `if os.getenv('PYTORCH_VERSION')` and `if ('IGNORE_TORCH_VER')`

Reply:
Added this; let me know if you also want to remove the PYTORCH_VERSION part!